### ISSUE TITLE
[delete] suspend and resume running instances when deleting

### ISSUE THREAD

**Issue by Saviq** [canonical/multipass#549]
We're currently shutting down instances to be deleted, even if they're running. We could instead suspend them so that when recovering, the instance is back in the state it was deleted at, ensuring no data loss.

Whether to resume on `multipass recover` I'm not sure yet.

**Comment by levkropp** [comment]
One important thing to look into with regards to this is if suspending an instance is faster than shutting it down, which is something that needs to be verified.
#138 is another related issue that discusses how `multipass delete` should behave with a suggestion for a `--wait` parameter to allow for a "slow" delete of an instance.

**Comment by ricab** [comment]
I don't think we regard the instance trash (vs straight delete) a very valuable feature these days. It is not present in the GUI and we have contemplated removing it entirely. We could choose to keep it and implement this, but I am lowering priority.
